engine/flink/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/source/flink/FlinkKafkaSource.scala

```diff
@@ -72,12 +77,62 @@ class FlinkKafkaSource[T](
 
   protected lazy val topics: NonEmptyList[TopicName.ForSource] = preparedTopics.map(_.prepared)
 
+  private val defaultOffsetResetStrategy = kafkaConfig.defaultOffsetResetStrategy.getOrElse(OffsetResetStrategy.None)
```

Comment:
Let's add annotation here:
  `private val defaultOffsetResetStrategy: OffsetResetStrategy` - you will see that
  kafkaConfig.defaultOffsetResetStrategy has type org.apache.kafka.clients.consumer.OffsetResetStrategy